keyboards/cannonkeys/practice60/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
To be honest, LTO isn't really needed here.  It's present on the AVR boards due to firmware size limitations, and I don't think that the F103 has the same issue.  upas/awkannan or somebody more knowledgeable about the hardware would have to confirm.